### DISCUSSION TITLE
spool tutorial for reading a folder

### DISCUSSION THREAD

**Discussion by jinwar** [DASDAE/dascore#80] [Ideas]
In the spool tuturial, can we add an example to use a folder to create a spool?
https://dascore.org/tutorial/spool.html

@d-chambers

**Comment by eileenrmartin** [comment]
Great idea! I think this might be a good thing to add as a to-do card in the [scrum board](https://github.com/orgs/DASDAE/projects/2)

**Comment by d-chambers** [comment]
Ya good idea, I overlooked that one.

There is such an example in the [quickstart](https://dascore.org/#index-a-directory-of-das-files) but there should be one in the spool section as well.

**Comment by d-chambers** [comment]
Added the spool tutorial stuff in #107 and #109
